README.md

```diff
@@ -831,24 +864,6 @@ link, pendingRef, err := descopeClient.Management.User().GenerateEnchantedLinkFo
 // Note 2: In case of testing sign-in / sign-up methods with test users, need to make sure to generate the code prior calling the sign-in / sign-up methods (such as: descopeClient.Auth.MagicLink().SignUpOrIn)
 ```
 
-## API Rate limits
```

Comment:
But how will develoeprs know what to do in such cases?

Reply:
Rate Limiting we removed from our website and also talked to @gshriki about this. Since our rate limit is so high, he thought we didn't really need our customers to check for this. I know most of our other competitors provide their rate-limiting information. If you think we should include the code, that may be fair, but then I'll just remove the link to the website that doesn't work anymore.

Reply:
Let's include the code and remove the link. Need to make sure to add to all sdks if missing.